.build/analyzers/nunit.test.step

```diff
@@ -24,6 +24,8 @@
   <target name="go" depends="run_tasks" />
 
   <target name="run_all_tests" depends="prepare, set_nunit_failure_mode, get_test_dlls_all, run_tests, generate_report" description="Like go but runs all tests" />
+  
+  <target name="run_full_tests" depends="prepare, set_nunit_failure_mode, get_test_dlls_full, run_tests, generate_report" description="Like go but runs a full set of the working tests" />

```

Comment:
Part of me wonders if this should be renamed to something like `run_integration_tests`, and narrow the scope of the test assemblies that it finds to only the ones with integration in the name.

This would mean that we could then choose to run all unit tests, and then all integration tests.

Thoughts?

Reply:
Hmm, that could work. Now that I am thinking about it, that would be good.

It would speed up the CI builds by a couple of seconds, as the unit tests are already run by `build.sh/cmd`, so they would not be run twice.

Reply:
This has been updated, hopefully it is working correctly.

Reply:
Ok, that is better, the casing was screwed up for the linux build.